### ISSUE TITLE
NativeEventEmitter Warnings since Upgrading with React Native 0.65

### ISSUE THREAD

**Issue by MEsteves22** [voximplant/react-native-voximplant#132]
Hello!

Since upgrading to React Native 0.65, I am having several warnings with NativeEventEmitter. The warnings I'm having are the following:

````
`new NativeEventEmitter()` was called with a non-null argument without the required `removeListeners` method. 
`new NativeEventEmitter()` was called with a non-null argument without the required `addListener` method. 
EventEmitter.removeListener('VIConnectionEstablished', ...): Method has been deprecated. Please instead use `remove()` on the subscription returned by `EventEmitter.addListener`.
EventEmitter.removeListener('VIAuthResult', ...): Method has been deprecated. Please instead use `remove()` on the subscription returned by `EventEmitter.addListener`.
````

These warnings seem to be located in the following files: Endpoint.js, Call.js, Client.js, AudioDeviceManager.js, AudioFile.js, Messenger.js

I am using `react-native-voximplant` version 1.25.0.

Thank you!

**Comment by wiredmatrix** [comment]
Am having the same warnings with AuthResult Error coder 491 whenever I try to login. Please any fix for this error?

**Comment by YuliaGrigorieva** [comment]
Hello!

We have recently released Voximplant React Native SDK 1.27.0 with the fix for these warnings.

Best regards,
Yulia Grigorieva
